src/main/webapp/scriptNzinga.js

```diff
@@ -0,0 +1,128 @@
+  cityDesc3Container.innerText = 'Jeju province encompasses the South' +
+      ' Korean island of Jeju in the Korea Strait. Its known for its' +
+      ' beachresorts and volcanic landscape of craters and cavelike' +
+      ' lava tubes.';
+}
+
+/**
+ * Adds Brazil info to page.
+ */
+function brazil() {
+  const headingContainer = document.getElementById('Heading');
+  headingContainer.innerText = 'Brazil';
+
+  const city1Container = document.getElementById('city1');
+  city1Container.innerText = 'Salvador';
+
+  const cityDesc1Container = document.getElementById('cityDesc1');
+  cityDesc1Container.innerText = 'Salvador, the capital of Brazil’s' +
+      ' northeastern state of Bahia, is known for its Portuguese colonial' +
+      ' architecture, Afro-Brazilian culture and a tropical coastline.';
+
+  const city2Container = document.getElementById('city2');
+  city2Container.innerText = 'Rio de Janeiro';
+
+  const cityDesc2Container = document.getElementById('cityDesc2');
+  cityDesc2Container.innerText = 'Rio de Janeiro is a huge seaside' +
+      ' city in Brazil, famed for its Copacabana and Ipanema beaches,' +
+      ' 38m Christ the Redeemer statue atop Mount Corcovado and for Sugarloaf' +
+      ' Mountain, a granite peak with cable cars to its summit.';
+
+  const city3Container = document.getElementById('city3');
+  city3Container.innerText = 'Jeju-do';
+
+  const cityDesc3Container = document.getElementById('cityDesc3');
+  cityDesc3Container.innerText = 'São Paulo, Brazil’s vibrant financial' +
+      ' center, is among the worlds most populous cities, with numerous' +
+      ' cultural institutions and a rich architectural tradition.';
+}
+
+/**
+ * Adds South Korea info to page.
+ */
+function southKorea() {
+  const headingContainer = document.getElementById('Heading');
+  headingContainer.innerText = 'South Korea';
+
+  const city1Container = document.getElementById('city1');
+  city1Container.innerText = 'Seoul';
+
+  const cityDesc1Container = document.getElementById('cityDesc1');
+  cityDesc1Container.innerText = 'Seoul, the capital of South Korea,' +
+      ' is a huge metropolis where modern skyscrapers, high-tech subways' +
+      ' and pop culture meet Buddhist temples, palaces and street markets.';
+
+  const city2Container = document.getElementById('city2');
+  city2Container.innerText = 'Busan';
+
+  const cityDesc2Container = document.getElementById('cityDesc2');
+  cityDesc2Container.innerText = 'Busan, a large port city in South' +
+      ' Korea, is known for its beaches, mountains and temples.';
+
+  const city3Container = document.getElementById('city3');
+  city3Container.innerText = 'Jeju-do';
+
+  const cityDesc3Container = document.getElementById('cityDesc3');
+  cityDesc3Container.innerText = 'Jeju province encompasses the ' +
+      'South Korean island of Jeju in the Korea Strait. Its known for' +
+      'its beach resorts and volcanic landscape of craters and cavelike' +
+      ' lava tubes.';
+}
```

Comment:
Please refactor this logic as it's pretty repetitive across all three functions. You might consider something like this:
```javascript
function malaysiaResults() {
  const headingContainer = document.getElementById('Heading');
  headingContainer.innerText = 'Malaysia';
  
  const city1 = {
      name: "Kuala Lumpur",
      description: "Description for Kuala Lumpur",
      id: 101,
  }
  const city2 = { /* city2 info */ }
  const city3 = { /* city3 info */ }
  displayResults(city1, city2, city3);
}

// similar brazilResults() and southKoreaResults() functions

function displayResults(city1, city2, city3) {
  // populate city1 info
  // populate city2 info
  // populate city3 info
}
```

Or even better (either in this PR or the next one), you could make `displayResults()` generic enough to accept any number of results and create the necessary HTML elements:
```javascript
function malaysiaResults() {
  const headingContainer = document.getElementById('Heading');
  headingContainer.innerText = 'Malaysia';
  
  const results = [{
      name: "Kuala Lumpur",
      description: "Description for Kuala Lumpur",
      id: 101,
  },
  // any additional cities
  ]

  displayResults(results);
}

// similar brazilResults() and southKoreaResults() functions

function displayResults(results) {
  for (let result in results) {
    // Create a row div if necessary or grab last row
    // Create a column div
    // Create a card div
    // Create the elements that go inside the card div
    // populate city information
    // append the information elements to the card div
    // append the card div to the column div
    // append the column div to the row div
  }
}
```
The latter option is closer to what's needed for the final product. For additional help I suggest taking a look at your comments feature from week 2 and also taking a look at the [Todo List](https://github.com/google/software-product-sprint/blob/master/walkthroughs/week-2-server/examples/todo-list/src/main/webapp/script.js) example from week 2.